rust-toolchain

```diff
@@ -1 +1 @@
-1.52.1
+nightly-2021-05-06
```

Comment:
We have some compile error that are not resolved in `nightly-2021-05-06`.

Reply:
yeah, I have the same problem in my dev env, especially when using the `cargo watch xxx` tool, the error you mentioned keeps popping out.

Since that, I switch to stable toolchain 1.52.1 during daily dev.

But I am not sure if that's appropriate (in case somebody or tools is using nightly features). So I  switch it back in the last commit.

Any suggestions?

Reply:
@BohuTANG  I've seen a gist that you created that keeps a similar error message, any idea about it?

Reply:
Sorry for the later, grcov nightly Rust is required:
https://github.com/mozilla/grcov

But we can try to rollback to a `stable` nightly version.

Reply:
Got it, nightly is ok.

Reply:
FYI, the master rust toolchain has change to nightly-2021-06-01, seems more stable.